services/clusters-keeper/src/simcore_service_clusters_keeper/utils/clusters.py

```diff
@@ -0,0 +1,13 @@
+from ..core.settings import ApplicationSettings
+
+
+def create_startup_script(app_settings: ApplicationSettings) -> str:
+    return "\n".join(
+        [
+            "git clone --depth=1 https://github.com/ITISFoundation/osparc-simcore.git",
+            "cd osparc-simcore/services/osparc-gateway-server",
```

Comment:
MINOR: if not too cumbersome, add a test that verifies that all reference here (i.e. paths and repo urls) are up-to-date

Reply:
maybe later. if I manage to remove the dask-gateway, this will go.